src/include/miopen/conv/invokers/impl_gemm_dynamic.hpp

```diff
@@ -219,19 +219,17 @@ ComputeDynamicIGemmForwardKernelArgs<solver::TunableImplicitGemmGTCDynamic_t>(
     opArgs.emplace_back(pad_w);
     opArgs.emplace_back(y);
     opArgs.emplace_back(x);
-    if(conv_problem.IsFp16())
-    {
-        opArgs.emplace_back(group);
-        opArgs.emplace_back(magic_0);
-        opArgs.emplace_back(magic_1);
-        opArgs.emplace_back(magic_2);
-        opArgs.emplace_back(magic_3);
-        opArgs.emplace_back(magic_4);
-        opArgs.emplace_back(magic_5);
-        opArgs.emplace_back(magic_6);
-        opArgs.emplace_back(shift_pack_0);
-        opArgs.emplace_back(shift_pack_1);
-    }
+    opArgs.emplace_back(group);
```

Comment:
[Q] Why this is necessary?

Reply:
Because this fast division method is also applicable for fp32 kernels.

Reply:
@shaojiewang Was it applicable before this PR?

Reply:
@atamazov no. I just make it be the same way as fp16 solver.